examples/build.gradle

```diff
@@ -34,4 +34,10 @@ dependencies {
     compile rootProject //TODO: Hack for now since in code freeze
     testCompile group: 'junit', name: 'junit', version: '4.12'
     testCompile 'org.hamcrest:hamcrest-library:1.3'
+
+    implementation('com.radixdlt:radixdlt-java-common') {
```

Comment:
Shouldn't need this here, as it is API in ../build.gradle.

Reply:
fixed